src/components/pages/request-form/request-form.component.js

```diff
@@ -0,0 +1,160 @@
+import Input from '@material-ui/core/Input';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+import useStyles from './request-form.styles';
+
+const RequestForm = () => {
+  const classes = useStyles();
+  const [values, setValues] = React.useState({
+    age: '',
+    name: 'hai',
+  });
+  function handleChangeB(event) {
+    setValues(oldValues => ({
+      ...oldValues,
+      [event.target.name]: event.target.value,
+    }));
+  }
+  const names = [
+    'Tom',
+    'Anna',
+    'Daniel',
+    'Cailey',
+  ];
+  const [personName, setPersonName] = React.useState([]);
+
+  function handleChange(event) {
+    setPersonName(event.target.value);
+  }
+  const [selectedDate, handleDateChange] = React.useState(new Date());
+  const [selectedDateB, handleDateChangeB] = React.useState(new Date());
+
+  return (
+    <Paper className={classes.paper}>
+      <Grid container spacing={3}>
+        <Grid
+          item
+          xs={12}
+          md={6}
+        >
+          <KeyboardDatePicker
+            className={classes.fullWidth}
+            required
+            autoOk
+            variant="inline"
+            label="Start Date"
+            format="MM/DD/YYYY"
+            value={selectedDate}
+            onChange={date => handleDateChange(date)}
+          />
+        </Grid>
+        <Grid
+          item
+          xs={12}
+          md={6}
+        >
+          <KeyboardDatePicker
+            className={classes.fullWidth}
+            autoOk
+            required
+            variant="inline"
+            label="End Date"
+            format="MM/DD/YYYY"
+            value={selectedDateB}
+            onChange={date => handleDateChangeB(date)}
+          />
+        </Grid>
+        <Grid
+          item
+          xs={12}
+          className={classes.padding12}
+        >
+          <FormControl fullWidth>
+            <InputLabel>Send request to</InputLabel>
+            <Select
+              multiple
+              required
+              value={personName}
+              onChange={handleChange}
+              input={<Input id="select-multiple-chip" />}
+              renderValue={selected => (
+                <div>
+                  {selected.map(value => (
+                    <Chip key={value} label={value} />
+                  ))}
+                </div>
+              )}
+            >
+              {names.map(name => (
+                <MenuItem key={name} value={name}>
+                  {name}
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+        </Grid>
+        <Grid
+          item
+          xs={12}
+          className={classes.padding12}
+        >
+          <FormControl className={classes.fullWidth}>
+            <InputLabel>Holiday Type</InputLabel>
+            <Select
+              value={values.age}
+              required
+              onChange={handleChangeB}
+              inputProps={{
```

Comment:
You can remove the input props property.

Reply:
Without that there will be nothing inside the select menu after we click on one of the options.